packages/editor/src/bindings/post-meta.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * WordPress dependencies
+ */
+import { useEntityProp } from '@wordpress/core-data';
+import { select } from '@wordpress/data';
+/**
+ * Internal dependencies
+ */
+import { store as editorStore } from '../store';
+
+const { getCurrentPostType } = select( editorStore );
```

Comment:
It feels like this should be moved to `useSource` to ensure it always triggers rerenders whenever the post type changes.

Reply:
Okay, I'll move it there 🙂

Reply:
Done in [this commit](https://github.com/WordPress/gutenberg/pull/58085/commits/478f861c2f7a5ca360617ee9bb40b7aa7f87d22d).